tests/commonTemplates_test.go

```diff
@@ -254,37 +249,17 @@ var _ = Describe("Common templates", func() {
 						testFlavorLabel:                      "true",
 						testWorkflowLabel:                    "true",
 					},
-					OwnerReferences: []metav1.OwnerReference{{
-						APIVersion: templatev1.GroupVersion.String(),
-						Kind:       "Template",
-						Name:       ownerTemplate.Name,
-						UID:        ownerTemplate.UID,
-					}},
 				},
 			}
+			Expect(libhandler.SetOwnerAnnotations(ssp, oldTemplate)).To(Succeed())
+
 			Expect(apiClient.Create(ctx, oldTemplate)).ToNot(HaveOccurred(), "creation of dummy old template failed")
 		})
 
 		AfterEach(func() {
 			Expect(apiClient.Delete(ctx, oldTemplate)).ToNot(HaveOccurred(), "deletion of dummy old template failed")
-			Expect(apiClient.Delete(ctx, ownerTemplate)).ToNot(HaveOccurred(), "deletion of dummy owner template failed")
 		})
 
-		It("[test_id:5621]should replace ownerReference with owner annotations for older templates", func() {
```

Comment:
Not required anymore?

Reply:
No. This test was for backward compatibility with a very old version of ssp-operator, before it was written in golang. Now none of the templates can have `ownerReferences` pointing to SSP, because they are in differetnt namespace than SSP.

Reply:
ACK